src/Middleware/OutputCaching/src/LoggerExtensions.cs

```diff
@@ -49,4 +49,10 @@ internal static partial class LoggerExtensions
         EventName = "ExpirationExpiresExceeded")]
     internal static partial void ExpirationExpiresExceeded(this ILogger logger, DateTimeOffset responseTime);
 
+    [LoggerMessage(12, LogLevel.Error, "Unable to query output cache.", EventName = nameof(UnableToQueryOutputCache))]
```

Comment:
Don't use `nameof` for the `EventName`. We treat `EventName` sort of like public API and using `nameof` means the `EventName` will change if we change the function name.

Reply:
noted; [changing to hard code](https://github.com/dotnet/aspnetcore/pull/49291/commits/96bd5e6da3b26bab0c84ca0673b2b71623bb8870)